integration/templates/next-app-router/package.json

```diff
@@ -12,7 +12,7 @@
     "@types/node": "^18.19.33",
     "@types/react": "18.3.3",
     "@types/react-dom": "18.3.0",
-    "next": "^13.5",
+    "next": "^14.1.1",
```

Comment:
⛔ We should drop this change since we want to run the tests with our minimum supported `next` version which is `13.5`